packages/feature-detector/scripts/patch-type-definition.ts

```diff
@@ -0,0 +1,106 @@
+// assumes being ran in packages/feature-detector
+
+import { get } from 'node:https'
+import { writeFile } from 'node:fs/promises'
+import { join } from 'node:path'
+
+const repositoryBaseUrl =
```

Comment:
maybe you want to fetch as `posinstall` script?

Reply:
Not really: users of this package will never run this script. This is only for publishers of this package: it has to be done as part of the build command.